vms/platformvm/txs/fee/complexity.go

```diff
@@ -72,6 +77,11 @@ const (
 		wrappers.IntLen + // deactivation owner threshold
 		wrappers.IntLen // deactivation owner num addresses
 
+	intrinsicBLSAggregateCompute = 5     // BLS public key aggregation time is around 5us
+	intrinsicBLSVerifyCompute    = 1_000 // BLS verification time is around 1000us
```

Comment:
by the way I just realized the verifying a PoP also involves un-compressing a compressec EC key and an EC signature. 

I guess we could say 1ms is good enough for both of them, but it's worth to mention it in code review since we also count the aggregation (which is linear, but still...)

Reply:
Added the time to decompress the key, the signature decompression time should be considered part of the signature verification time :+1:.